lms/static/js/fixtures/instructor_dashboard/batchenrollment.html

```diff
@@ -0,0 +1,71 @@
+<section id="membership" class="idash-section active-section" aria-labelledby="header-membership">
+    <h2 class="hd hd-2" id="header-membership">Membership</h2>
+
+    <fieldset class="batch-enrollment membership-section">
+        <legend id="heading-batch-enrollment" class="hd hd-3">Batch Enrollment</legend>
+        <label>
+            Enter email addresses and/or usernames separated by new lines or commas.
+            You will not get notification for emails that bounce, so please double-check spelling.
+            <textarea rows="6" name="student-ids" placeholder="Email Addresses/Usernames" spellcheck="false"></textarea>
+        </label>
+        <input id="is_course_white_label" value="False" type="hidden">
+        <div class="enroll-option">
+            <label for="course-modes-list-selector" class="has-hint">Select a course mode:
+                <div class="hint auto-enroll-hint">
+                    <span class="hint-caret"></span>
+                    <p id="auto-enroll-tip">
+                        The users will be automatically enrolled in the <em>selected</em> course mode.
+                        <br><br>
+                        Selecting from course modes has no effect if 'Unenroll' is selected.
+                    </p>
+                </div>
+            </label>
+            <select id="course-modes-list-selector" name="course_mode" class="course-modes-list-selector">
+                <option value="honor">Honor Certificate</option>
+                <option value="verified">Verified Certificate</option>
+            </select>
+        </div>
+        <div class="enroll-option">
+            <label class="has-hint">
+                <input name="auto-enroll" id="auto-enroll" value="Auto-Enroll" checked="checked"
+                       aria-describedby="heading-batch-enrollment" type="checkbox">
+                <span class="label-text">Auto Enroll</span>
```

Comment:
Also this change will have a greater scope since we will also need to update the existing documentation http://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/manage_live_course/course_enrollment.html#options-for-enrolling-learners-in-a-course